DQMOffline/Trigger/python/HiggsMonitoring_Client_cff.py

```diff
@@ -427,8 +427,40 @@
     ),
 )
 
+ggHbbEfficiency = DQMEDHarvester("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/ggHbb/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "effic_metME       'efficiency vs MET; MET [GeV]; efficiency' metME_numerator       metME_denominator",
+        "effic_jetEta_1       'efficiency vs leading jet eta; jet eta ; efficiency' jetEta_1_numerator       jetEta_1_denominator",
```

Comment:
do you need this ?
I think you have the variable binning one as well

Reply:
I don't have a variable binning jetEta histo. I'll remove that efficiency curve.